core/client/fs/src/main/java/alluxio/client/block/stream/GrpcDataMessageBlockingStream.java

```diff
@@ -58,10 +58,15 @@ public GrpcDataMessageBlockingStream(Function<StreamObserver<ResT>, StreamObserv
 
   @Override
   public ResT receive(long timeoutMs) throws IOException {
+    return retryReceive(timeoutMs, null);
+  }
+
+  @Override
+  public ResT retryReceive(long timeoutMs, ReqT request) throws IOException {
```

Comment:
similar comment about whether a separate `retryReceive` is necessary if `receive` directly calls it.